### DISCUSSION TITLE
Toggle Label Text in GUI

### DISCUSSION THREAD

**Discussion by jmdelahanty** [talmolab/sleap#776] [Help!] [answered]
Hello SLEAP-y heads!

I was just chatting with a member of the Tye Lab and they were telling me about how it can be hard to move labeled points sometimes because the text of labels overlap one another.

Here's an example:
![Sleap Label Overlap](https://user-images.githubusercontent.com/28411193/171958734-933bbf7e-c441-4e7b-aa45-d1f9148575b1.PNG)

You can zoom in of course and the text won't overlap much most of the time, but he said that it would be cool to have a way to toggle the text on/off in the GUI. He also said sometimes with a juvenile mouse the labels can nearly completely obfuscate it since they're so small! If all that was shown was the points then he wouldn't have that problem as much maybe.

**Comment by talmo** [comment, marked as answer]
Hi @jmdelahanty,

This is available from the GUI already through **View** → **Show Node Names** (Ctrl + Tab by default).

**Reply by jmdelahanty** [reply]
Very cool! Now they know and are pretty happy about it.
